tools/obscuroscan_v3/frontend/api/batches.ts

```diff
@@ -0,0 +1,41 @@
+import { httpRequest } from ".";
+import { apiRoutes } from "@/src/routes";
+import { pathToUrl } from "@/src/routes/router";
+import { ResponseDataInterface } from "@/src/types/interfaces";
+import {
+  Batch,
+  BatchDetails,
+  BatchResponse,
+} from "@/src/types/interfaces/BatchInterfaces";
+
+export const getBatches = async (
+  payload?: Record<string, any>
+): Promise<ResponseDataInterface<BatchResponse>> => {
+  const data = await httpRequest<ResponseDataInterface<BatchResponse>>({
+    method: "get",
+    url: pathToUrl(apiRoutes.getBatches),
+    searchParams: payload,
+  });
+  return data;
+};
```

Comment:
These seem to be methods created to be imported into other parts of the code.
It might be confusing as `getBatches` is a pretty common name. What if it was something like `getBatchesAPI`.